kuksa_apps/kanto/edge_client.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) 2022 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
```

Comment:
Please, update the **copyright headers** to the **Apache 2.0** ones of the kuksa.val project.

Reply:
I updated the headers according to some examples in the kuksa.val project.